inout/eventtypes.go

```diff
@@ -0,0 +1,100 @@
+package inout
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"io"
+
+	svix "github.com/svix/svix-libs/go"
+)
+
+func ImportEventTypesJson(sc *svix.Svix, reader io.Reader) error {
+	dec := json.NewDecoder(reader)
+	var eventTypes []*svix.EventTypeIn
+	err := dec.Decode(&eventTypes)
+	if err != nil {
+		return err
+	}
+	for _, et := range eventTypes {
+		err = createOrUpdateEventType(sc, et)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func ImportEventTypesCsv(sc *svix.Svix, reader io.Reader) error {
+	csvReader := csv.NewReader(reader)
+	for {
+		record, err := csvReader.Read()
+		if err == io.EOF {
+			break
+		}
+		if err != nil {
+			return err
+		}
+		if len(record) < 2 {
+			return fmt.Errorf("invalid csv record")
+		}
+		et := &svix.EventTypeIn{
+			Name:        record[0],
+			Description: record[1],
+		}
+		err = createOrUpdateEventType(sc, et)
```

Comment:
Should we add a `--force` flag and only fallback to updating the eventType if `--force=True`. Otherwise if `--force=false` just fail when the eventType already exists?

Reply:
This is a good suggestion. Just to add: probably want to skip existing event types, not fail.